third_party/gpus/crosstool/CROSSTOOL.tpl

```diff
@@ -121,6 +121,9 @@ toolchain {
   # linker_flag: "-Wl,--warn-execstack"
   # linker_flag: "-Wl,--detect-odr-violations"
 
+  # Include directory for cuda headers.
+  cxx_builtin_include_directory: "/usr/local/cuda%{cuda_version}/include"
```

Comment:
In think that should be enough. Using _get_cxx_inc_directories is definitely safer.